src/main/resources/api/openapi.yaml

```diff
@@ -1245,3 +1279,71 @@ paths:
             text/plain; charset=UTF-8:
               schema:
                 type: string
+
+  /wallet/assets/issue/generate:
```

Comment:
Maybe we could use just `/wallet/assets/generate`

Reply:
It is supposed that each of these routes relates to transaction creation - the differences are the action that will be taken after (send transaction or not) and the actual type of transaction ("asset issue" or "payment")